src/ORM/DatabaseAdmin.php

```diff
@@ -292,10 +299,21 @@ public function doBuild($quiet = false, $populate = true, $testMode = false)
 
                 // Log data
                 if (!$quiet) {
+                    if ($showRecordCounts && DB::get_schema()->hasTable($tableName)) {
+                        try {
+                            $count = DB::query("SELECT COUNT(*) FROM \"$tableName\"")->value();
+                            $countSuffix = " ($count records)";
+                        } catch (Exception $e) {
+                            $countSuffix = " (error getting record count)";
+                        }
+                    } else {
+                        $countSuffix = "";
+                    }
+
                     if (Director::is_cli()) {
-                        echo " * $tableName\n";
+                        echo " * $tableName$countSuffix\n";
```

Comment:
Wouldn't it make more sense to have the space here (and below), instead of in the variable?

Reply:
TBH, I'm happy with this - if you want to PR after it's merged to change it, that's fine, but I don't have a problem with it as is